zitadel/v2/login_policy/resource.go

```diff
@@ -34,6 +34,11 @@ func GetResource() *schema.Resource {
 				Required:    true,
 				Description: "defines if a user MUST use a multi factor to log in",
 			},
+			forceMFALocalOnlyVar: {
+				Type:        schema.TypeBool,
+				Required:    true,
+				Description: "if activated, only local authenticated users are forced to use MFA. Authentication through IDPs won't prompt a MFA step in the login.",
```

Comment:
see default_login_policy

```suggestion
				Description: "if activated, ZITADEL only enforces MFA on local authentications. On authentications through MFA, ZITADEL won't prompt for MFA.",
```